azerothcore/azerothcore-wotlk
# fix(DB/creature): Remove money drops from Imp Minion

## Changes Proposed:
Summoned [Imp Minions](https://tbc.wowhead.com/npc=12922/imp-minion) currently drop 1-53 copper, which is incorrect. This PR removes their money drops. 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8750
- Closes https://github.com/chromiecraft/chromiecraft/issues/1858

## SOURCE:

Imps shouldn't drop coins. Any NPC who does will have a 'Worth' field on Wowhead giving average coin drops. Imps do not have this listed at https://tbc.wowhead.com/npc=12922/imp-minion

## Tests Performed:

- Ran SQL, no errors/warnings, 1 row changed as expected.
- checked in game, minion is no longer lootable.

## How to Test the Changes:


1. Check in Keira.
2. Alternatively, .go c 6433
3. Kill the cultist and its minion
4. Observe minion is not lootable.

## Known Issues and TODO List:


N/A.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
